frontend/src/metabase/visualizations/shared/settings/cartesian-chart.ts

```diff
@@ -210,6 +210,7 @@ export const getDefaultLegendIsReversed = (
 
 export const getDefaultShowDataLabels = () => false;
 export const getDefaultDataLabelsFrequency = () => "fit";
+export const getDefaultMaxCategories = () => 8;
```

Comment:
I assume we should not have such default because it affects already existing content. Previously created, reviewed, shared, approved charts of customers will appear with the grouping. To prevent this there are a couple of options:
- Do not have the default — seems reasonable to me
- Migrate already existing cards so that they have high enough value that do not unintentionally change the charts